pkg/config/struct_keys_test.go

```diff
@@ -60,7 +60,7 @@ func TestStructKeys_SimpleTagged(t *testing.T) {
 	type s struct {
 		A int `test:"Aaa"`
 		B int `toast:"bee"`
-		c int `test:"ccc" toast:"sea"`
```

Comment:
I think this is intentional.

Reply:
Do you think it was to see if we can extract the metadata from the struct for private fields - because usually the linter things we are not using the field.

Reply:
@arielshaqed this is by design? if so I'll revert and leave a comment